src/_i18n/en.yml

```diff
@@ -629,3 +629,13 @@ projects:
                      <p>Part of the feature set included a reporting tool to provide the business with detailed reports on the event. We spent time to help train the AWS marketing team on how to use the system.</p>
     outcome: <p>Given the tight feedback loops and real-time interactions with the Product Owner, we were able to deliver both the MVPs for the project and several additional features ahead of the deadline. The event team were able to respond quickly to demand via the real-time data analysis (for instance session scheduling, staff rotas) and report session take-up and popularity to stakeholders.</p>
              <p>AWS invited Codurance to add additional features to the system so that it could be used at future events. Our team also gave a talk on Software Craftsmanship at the AWS Summit London 2016.</p>
+  MANDS:
+    link-text: Case study at M&S
+    short_description: We helped transition their processes to cater for new projects
```

Comment:
I just realised on the company page that we have a mix of separators `-` vs `_` and is the same here

Reply:
Will take care of that as a separate branch/PR